IEZE/Messenger/Messenger.Persistence/Configurations/AttachmentConfiguration.cs

```diff
@@ -0,0 +1,43 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+using Messenger.Persistence.Entities;
+
+namespace Messenger.Persistence.Configurations
+{
+    public class AttachmentConfiguration : IEntityTypeConfiguration<Attachment>
+    {
+        public void Configure(EntityTypeBuilder<Attachment> builder)
+        {
+            builder.ToTable("Attachments");
+
+            builder.HasKey(a => a.AttachmentID);
+
+            builder.Property(a => a.MessageID)
+                   .IsRequired();
```

Comment:
ar tikrai siti reikalingi? modelyje jeigu MessageId value taip ir non nullable, tai by default it is required?